FinanceSchedulerDemo/SpecflowTests/Steps/LoginSteps.cs

```diff
@@ -0,0 +1,63 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using SpecflowTests.Pages;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using TechTalk.SpecFlow;
+using TechTalk.SpecFlow.Assist;
+
+namespace SpecflowTests.Steps
+{
+    [Binding]
+    public sealed class LoginSteps
+    {
+        LoginPage loginPage = null; // DI it also
```

Comment:
DI this class into the constructor

Reply:
still not found the way to DI login page